core/services/ocr2/plugins/directrequestocr/aggregation.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 func CanAggregate(N int, F int, observations []*ProcessedRequest) bool {
-	return N > 0 && F >= 0 && len(observations) > 0 && len(observations) <= N && len(observations) >= N-F
+	return N > 0 && F >= 0 && len(observations) > 0 && len(observations) <= N && len(observations) >= 2*F+1
```

Comment:
N>=3F+1 therefore  N-F >= 2F+1
I.e., N-F is better because it also supports configurations where N>3F+1, for example N=5 and F=1

Reply:
@stchrysa I'm confused. I thought we want a weaker check here, no?

Quotes from Lorenz from our chat group:
"Please modify the check: we require at least 2*f + 1 observations"
"That's what's enforced on the OCR side, and since 2*f+1 <= n-f, you could run into liveness issues if the check remains as is"

Reply:
My bad, I though that OCR had also N-F as a check.